metrics/web/docs/features/sentry/diagrams/sentry_util_class_diagram.puml

```diff
@@ -0,0 +1,20 @@
+@startuml sentry_util_class_diagram
+package metrics {
+    class MetricsApp {}
+
+package util {
+    class SentryUtil {
+      + Future<void> init(String dsn, String releaseName, AppRunner appRunner, bool isFlutterErrorsEnabled)
+      + Future<void> captureException(dynamic throwable, dynamic stackTrace)
```

Comment:
```suggestion
      + Future<void> init(String dsn, String releaseName, AppRunner appRunner, bool isFlutterErrorsEnabled)
      + Future<void> captureException(Object throwable, Object stackTrace)
```
Maybe `Object` is better as we`re trying to avoid using the `dynamic` in usual methods?